pymeasure/instruments/etslindgren/emcenter.py

```diff
@@ -0,0 +1,53 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+
+import logging
+
+from pymeasure.instruments import Instrument
+
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class EMCenter(Instrument):
```

Comment:
The purpose of this class is to implement the "LOCAL" command and `reboot` method only. Is this correct ?
Is there any link between main frame and sub-modules ?

For the order of elements in a class, I would suggest to use the one mentioned here:
https://github.com/pymeasure/pymeasure/pull/535#issuecomment-1009202829

Reply:
I'm guessing this is WIP and will still get the modular card class instances (like channels)?

Reply:
Thank you for the suggestions and comments. I've made most of the changes. 
Can you give me some more details about what you mean by modular card class instances? How this "instrument" works is that it is a linux box that has intefaces for control (LAN/USBSERIAL/GBIP<I think it might be able to control other intr though this too>). It also Has expantion slots on the back for the modular cards (controlled by <slot><sub-card device id><command>).
Right now I control it without using the mainframe itself and just connecting "directly" to the card.
I also only have a positional card onhand, and have implemented most of the basic features that I have equipment for. 
The ones I haven't implement are mainly used for inital setup and wont be used often in normal op, but I can implement them if desired
Thanks